### PR TITLE
added plus 1 for each iteration in find destination

### PR DESCRIPTION
**Pull request by LiorQM**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Cisco-umbrella-cloud-security/ReleaseNotes/2_0_2.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Cisco Umbrella Cloud Security v2
+- Updated pagination iterations
```
Comment (ShirleyDenkberg) [inline Packs/Cisco-umbrella-cloud-security/ReleaseNotes/2_0_2.md:3]:
```suggestion
Updated pagination iterations
```

**Comment by ShirleyDenkberg** [comment]
@RotemAmit Doc review completed.

**Review by RotemAmit — COMMENTED** [review]
Nice work!
* Please update the docker image and the release notes file accordingly.

**Review by RotemAmit — APPROVED** [review]
Thank you very much for the quick fix!

**Pull request merged**